packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,199 @@
+
+The second SVG element with the "WaferMapArea" class attribute represents the area where the dies get rendered.
+The rectangle present here is used as the "highlight" which allows us to identify individual dies in the canvas.
+
+- Slot Names: (none)
+- Host Classes: (none)
+- Slotted Content/Slotted Classes: (none)
+- CSS Parts: (none)
+
+### Angular integration
+
+The `nimble-wafer-map` component does not take user input directly from the webpage where it's embedded. All the inputs must be provided through the constructor of the component so in the current case there is no point using ControlValueAccessors.
+
+### Blazor integration
+
+Blazor support is out of scope for the current implementation.
+
+### Visual Appearance
+
+For the initial component, we are going with the original wafer map appearance. 
+![WaferMap Example](file:///Resources/wafer_map_example.png)
+
+## Implementation
+
+As mentioned above, this is not a component that we start from scratch, but a visualization that already exists which is outdated and not futureproof. 
+As a first step, we are working on the "bloat removal" which is basically the removal of unwanted but exiting features (legend, tooltip etc.) and dependencies.
+
+Original component: [wafermap](https://dev.azure.com/ni/DevCentral/_git/op-web-portal?path=/libraries/wafermap&version=GBfeature/microServicesSplit)\
+Once the "bloat removal" is finished, the next step is to adapt the current 
+
+### Code submission strategy
+
+In order to make the process of submitting the re-written wafer map component easier for us and for the code reviewers as well, we propose the following bottom-up approach:
+* As an initial step, we submit a skeleton of the `nimble-wafer-map` component into the main branch. This will serve as a starting point on which the complete functionality will be built on.
+* Once the skeleton is in place, we start to work on and submit individual pieces of the functionality so it's not very overwhelming to review the code.
+
+### States
+
+We are not planning to allow the user to generate an “empty" wafer map, so we must ensure that data is provided when instantiating a new `nimble-wafer-map` component.
+The only action which we could consider as a "state transition" is the zoom functionality which was specified in the sections above.
+We expect that the end user be able to zoom in on the dies withing the wafer map and be able to zoom out by using the mouse wheel.
+Also, the user must be allowed to pan the zoomed in wafer map into any direction of its XY axis by holding the left mouse button and moving the mouse cursor around.
+
+Another interaction, which must be part of the component is the ability to identify the location of the mouse pointer on the wafer map die area.
+
+### Globalization
+
+Currently there will be no string data presented by this component. Might change in the future, but for now it will contain only numerical data.
```

Comment:
```suggestion
Currently there will be no string data presented by this component. Might change in the future, but for now it will contain only numerical data. Similar to the `nimble-number-field`, initially we don't plan to support locale-specific numeric formatting.
```